ReactNativeClient/lib/components/ModalDialog.js

```diff
@@ -22,12 +22,12 @@ class ModalDialog extends React.Component {
 				justifyContent: 'center',
 			},
 			modalContentWrapper: {
-				flex: 1,
+				height: '90%',
+				...Platform.select({ ios: { marginVertical: 40, marginHorizontal: 20 }, android: { margin: 20 } }),
```

Comment:
This is not a platform-specific, but device-specific issue (some Android phones also have a notch, and not all iPhones have one). To keep it simple please set marginVertical to 40 in all cases, and please add a comment linking to this pull request.